README.md

```diff
@@ -2,26 +2,30 @@
   <a href="https://github.com/digicatapult/check-version/actions"><img alt="check-version status" src="https://github.com/digicatapult/check-version/workflows/build-test/badge.svg"></a>
 </p>
 
-# Compares versions in package.json and package-lock.json and check they are higher than the latest published tag
+# Compares versions in package.json and package-lock.json and check they are higher than the latest published tag also does for Cargo manager
 
-This action reads package.json and package-lock.json version property and asserts that they are the same. The action fails if they are not.
+This action can read package.json and package-lock.json as well as Cargo.toml version property and asserts that they are the same.
 
 The action then gets all tags, selects the newest one per semver rules and checks that your local version is higher than newest published tag.
 
 ## To use this action in your workflow, provide these inputs:
 
 ```
 with:
-  npm_package_location: './' // optional
+  package_location: './' // optional
   token: ${{ secrets.GITHUB_TOKEN }}
   fail_on_same_version: 'true' // optional
+  manager: 'cargo' // optional
+  names: 'node' // optional and subject to change
 ```
 
 | input                | required | default | description                                                                                                          |
 | :------------------- | :------: | :-----: | :------------------------------------------------------------------------------------------------------------------- |
-| npm_package_location |    N     |  `./`   | If `package.json` and `package-lock.json` are not in the root directory, provide the correct path to their location. |
+| package_location |    N     |  `./`   | If `package.json` and `package-lock.json` are not in the root directory, provide the correct path to their location. |
 | token                |    Y     |    -    | Provide `${{ secrets.GITHUB_TOKEN }}` so the action can access the GitHub API                                        |
 | fail_on_same_version |    N     | `true`  | Set whether the action should fail if the version exactly matches the latest published tag.                          |
+| manager              |    N     | `npm`  | Allows you to specify other package manager `cargo`.
+| names                |    N     | `true`  | Takes a package name. This is for mono repos mainly for `dscp-node`.                                        |
```

Comment:
I'm not a huge fan of this. By changing these parameters we would have to update every single repo that uses this action and it would be a major version bump. Can I recomend we keep `npm_package_location` and add `cargo_package_location` for the location of the single cargo package to consider.

Reply:
they haven't?